data/tlg2001/tlg043/tlg2001.tlg043.opp-grc1.xml

```diff
@@ -1,11167 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
```

Comment:
I am not sure about what has been altered this file. Could it be a merging conflict?